mpitests/test_apply.py

```diff
@@ -573,6 +619,7 @@ def test_expval_single_wire_no_parameters(self, tol, operation, wires):
         obs = operation(wires)
         expval_single_wire_no_param(tol, obs)
 
+    @pytest.fixture(params=[np.complex64, np.complex128])
```

Comment:
Actually, `mark.parametrize` doesn't work here. Using `fixture` to define a `device_gpu_mpi` method to create a device with MPI support  was my initial plan. The reason why I didn't use it is that methods like `probability` only support `FP64`. Since almost all methods are ready, I will create a new PR to update it later.